starter-code/memory.js

```diff
@@ -41,21 +42,109 @@ MemoryGame.prototype._shuffleCard = function() {
     this.Cards[counter] = this.Cards[index];
     this.Cards[index] = temp;
   }
-  return;
+  return this.Cards[index];
 };
 
+var memoryGame;
+$(document).ready(function(){
 
+  // create an object of MemoryGame and generate a card
+  memoryGame = new MemoryGame();
+  
+  // get all element which class is ".card"
+  var $cards = $('.card');
+  
 
+  // traverse the array of cards
+  for (var index = 0; index < $cards.length; index++) {
+    $($cards[index]).on('click', function(e) {
```

Comment:
You don't have to use a loop here.  Just `$(.card).on(click, someFunction)` would work just fine to attach the event listener to all of the cards.